cubequery/tasks/__init__.py

```diff
@@ -166,6 +180,105 @@ def ping_results(self, output_url, results):
 
     herd_avoidance_timeout = 60
     cache_duration = 60 * 60 * 24  # One day of seconds
+    
+    def standard_validation(self, args):
+        """
+        Validates conditions based upon the combination of the parameters provided.
+
+        Loads conditions set in input_conditions.json
+        
+        """
+        
+        _settings_json = fetch_form_settings()  
+        
+        if not _settings_json:          
+            with open('input_conditions.json') as res_json: 
+                _settings_json = json.load(res_json)  
+
+        keys = [k for k in _settings_json if k in args]
+
+        errors = []
+        
+        # Validates AOI
+        if 'aoi' in args:

```

Comment:
This should deal with any field that has WKT as its type, not just one magically named 'aoi' we can't be sure that the scientists using this will always call the location aoi. (they should but, we cant 100% be sure)